### ISSUE TITLE
can't end measure when position is same with Marker

### ISSUE THREAD

**Issue by Zhaojing0987** [ppete2/Leaflet.PolylineMeasure#59]
when add measure point, it will under Marker if position is same, so I can't double click the point to end it.
![image](https://user-images.githubusercontent.com/29828346/59344612-0b17a380-8d41-11e9-9620-eb44b85f7fc2.png)
it seems Marker's z-index is much higher than currentCircle, how can I make measure front

**Comment by ppete2** [comment]
There've been a similar issue [https://github.com/ppete2/Leaflet.PolylineMeasure/issues/24](url) where we could not set the measuring point onto a CircleMarker. Within this issue sombody provides a JSFiddle with a demo of the problem: [https://jsfiddle.net/northtree/0d14vgxy/](url)
As you can see this is not problem any more.

As for your issue: In the JSFiddle you could see, that the CircleMarkers are below the Measure points. So you have to make sure within your script that the z-index of the Marker is also below the measure points. There's nothing to modify within the PolylineMeasure addon.

**Comment by LeonardoRochaInacio** [comment]
Hi @zhaoo-jing,

Probably the problem on your situation is the same as mine: the CircleMarker class is rendered in the 'OverlayPane' and the Marker class is rendered in 'markerPane'.

As you can see here in the line 852 from Leaflet.PolylineMeasure.js
```javascript
var newCircleMarker = new L.CircleMarker (latlng, polylineState.options.currentCircle).addTo(polylineState._layerPaint);
``` 
All that put inside currentCircle options will be passed to CircleMarker constructor

The CircleMarker construction is the following:
```javascript
L.circleMarker(<LatLng> latlng, <CircleMarker options> options?)
``` 

In further details, the 'options' attribute can support an option named 'pane' inherited from class 'Layer', so you can take advantage of that to inform the correct pane.

In short, all you need to do when creating you polylineMeasure instance, is:
```javascript
L.control.polylineMeasure({
	startCircle: {                 
		pane: 'markerPane'                   
	},
	intermedCircle: {            
		pane: 'markerPane'                
	},
	currentCircle: {              
		pane: 'markerPane'
	},
	endCircle: {                   
		pane: 'markerPane'                 
	}
}).addTo(this.$refs.editorMap.leafletObject);
``` 
I know it's a little bit late, but hope it help you!
